src/util/StringUtils.h

```diff
@@ -381,6 +385,33 @@ vector<string> split(const string& orig, const char sep) {
   return result;
 }
 
+// _____________________________________________________________________________
+vector<string> splitWs(const string &orig) {
+  vector<string> result;
+  if (orig.size() > 0) {
+    size_t start = 0;
+    size_t pos = 0;
+    while (pos < orig.size()) {
+      if (isspace(orig[pos])) {
```

Comment:
This is `std::isspace()` from cctype, so if we use this would make sense to write the `std::` explicitly. Then again we may actually want to use `::isspace()` globally imported from `#include <ctype.h>` because unlike `std::isspace()` this doesn't depend on the locale and is thus probably a bit faster. Since we're dealing with what is basically always UTF-8 encoded but in the ASCII subset we don't really need locale support I think.

Reply:
That said, the C function seems to have its own pitfalls and requires an additional cast to [unsigned char](https://stackoverflow.com/questions/8966451/how-to-use-the-isspace-function-in-c#comment11248661_8966474).. So maybe we do want to use std::isspace()